pkg/controller/plan/kubevirt.go

```diff
@@ -312,6 +315,20 @@ func (r *KubeVirt) EnsureVM(vm *plan.VMStatus) (err error) {
 			err = liberr.Wrap(err)
 			return
 		}
+		if r.isOvirtProvider(vm) && pvc.Spec.DataSource.Kind == "OvirtVolumePopulator" {
```

Comment:
I'm not sure if there's a better way to do it but how about replacing those strings with something like reflect.TypeOf(OvirtVolumePopulator{}).Name() ?